homeassistant/components/permobil/strings.json

```diff
@@ -27,7 +27,8 @@
       "region_fetch_error": "Error fetching regions",
       "code_request_error": "Error requesting application code",
       "invalid_email": "Invalid email",
-      "invalid_code": "The code you gave is incorrect"
+      "invalid_code": "The code you gave is incorrect",
+      "unsigned_eula": "Please sign the EULA in the MyPermobil app"
```

Comment:
Maybe use a placeholder description for MyPermobil if that is a name that shouldn't be translated?

Reply:
Does this mean i should include something like 
`"unsigned_eula": "Please sign the EULA in the [%....%] app"`
and have the placeholder point to a string const which is equal to "MyPermobil"?